.github/workflows/release-bot.yml

```diff
@@ -0,0 +1,18 @@
+name: Pushes release updates to a pre-defined Matrix room
+on:
+  release:
+    types:
+      - edited
+      - prereleased
+      - published
```

Comment:
Only published status, if this is something that we want. (Not convinced)

Reply:
Ok, no problem, I can change it to that.